database/migrations/2020_05_10_015415_create_hh5p_libraries_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateHH5pLibrariesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('hh5p_libraries', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->timestamps();
+            $table->string('name', 127);
```

Comment:
Name can not have more letters by h5p or our decision? I would keep 255.

Reply:
https://h5p.org/content-types-and-applications names are short, 127 are enought. You can't change those names